content/docs/guide.md

```diff
@@ -44,9 +50,28 @@ built from the master branch in Copr:
     $ # OR in case you have packit already installed from the Fedora repositories:
     $ sudo dnf upgrade packit
 
+### From PyPI
+
+We publish packit to PyPI and it's available as `packitos` project — **packit
+at PyPI is something different**.
+
+    $ pip install --user packitos
+
+### From Source
+
 ... or installing it directly from GitHub:
 
-    $ pip3 install --user git+https://github.com/packit/packit
+    $ pip install --user git+https://github.com/packit/packit
+
+### In a container
+
+If none of the above work for you, try running it in a container
+from our Fedora based image. It contains packit installed from `main` branch,
+i.e. the same you'd get by pip installing from Github.
+
+    $ podman/docker run -ti --rm -v $(pwd):/src --security-opt label=disable quay.io/packit/packit bash
```

Comment:
```suggestion
    $ podman run -ti --rm -v $PWD:/src --security-opt label=disable quay.io/packit/packit bash
```
* we're Red Hat, we should advocate for podman where possible
* $PWD is faster since it doesn't run any syscalls